bylaws/100-implementation.md

```diff
@@ -1,17 +1,15 @@
-# Implementation
-
-These bylaws must be adopted by a 2/3 vote of all Voting Representatives, with a quorum of 50%.  Explicit abstentions (+0) will count toward quorum but do not count toward majority.
+# Implementation of FIG 3.0 changes
 
 Upon the adoption of these bylaws:
 
 All currently sitting Secretaries will remain in their positions with no change in their term.
 
-The recognized Voting Representatives for member projects have 30 days to state their intent to remain member projects.  If they do so, the Voting Representative will be recognized as the Project Representative going forward. No new admission vote is necessary.  If the Voting Representative does not make any statement for 30 days, the project will be dropped from FIG membership.
+The recognized Voting Representatives for member projects have 30 days to state their intent to remain member projects. If they do so, the Voting Representative will be recognized as the Project Representative going forward. No new admission vote is necessary. If the Voting Representative does not make any statement for 30 days, the project will be dropped from FIG membership.
 
-The Core Committee will be selected as though there were a 12-member vacancy according to the Vacancy rules above, including postponing the selection until the next regular election if appropriate.
+The Core Committee will be selected as though there were a 12-member vacancy according to the Vacancy rules above. The timetable of when this vote will occur will be announced by the Secretaries.
 
 All previously-approved PSRs will remain approved with no change, and will retain their current PSR number.
 
-The Editor of any PSR that is in Draft or Needs Review state will have 30 days from the time the Core Committee is seated to form an official Working Group, as defined in this document.  If a 5-member working group is formed then the PSR will automatically continue under these rules with no need for re-approval or re-entry.  If the 5-member Working Group cannot be formed in that time then the PSR will automatically be listed as Abandoned.  The counter for when a PSR proposal is inactive and subject to automatic abandonment starts only when the Working Group is formally defined.
+The Editor of any PSR that is in Draft or Needs Review state will have a minimum of 30 days, and a maximum of 90 days, from the time the Core Committee is seated to form an official Working Group, as defined in these bylaws. If a 5-member working group is formed then the PSR will automatically continue under these rules with no need for re-approval or re-entry. If the 5-member Working Group cannot be formed in that time then the PSR will automatically be listed as Abandoned. The exact timetable will be announced by the Secretaries.
```

Comment:
It's more I don't think we should define the exact timing in the bylaw; I don't want to see active specs kicked out because of technicalities. Removed the first bit.